modules/candidate_list/help/candidate_list.md

```diff
@@ -18,7 +18,7 @@ For example, searching with "531" under "DCCID" will retrieve the profile of can
 
 ## Option 3: Advanced Filter Options
 
-More detailed filter options can be accessed by clicking on the "Advanced" button. Users will select from a number of drop-down select boxes including Site, Project, Subproject, Gender etc., and then click "Show Data" to view a list of candidates. To return to "Basic" selection filters please select "Basic".
+More detailed filter options can be accessed by clicking on the "Advanced" button. Users will select from a number of drop-down select boxes including Site, Project, Subproject, biological sex, etc., and then click "Show Data" to view a list of candidates. To return to "Basic" selection filters please select "Basic".
```

Comment:
minor request but could we get a capital B for 'biological sex' here? thanks

Reply:
done